plugin/core/handlers.py

```diff
@@ -20,6 +21,14 @@ def config(self) -> ClientConfig:
     def additional_variables(cls) -> Optional[Dict[str, str]]:
         return None
 
+    @classmethod
+    def storage_path(cls) -> str:
+        """
+        The storage path. Use this as your base directory to install server files. Its path is '$DATA/Package Storage'.
+        You should have an additional subdirectory preferrably the same name as your plugin.
```

Comment:
s/plugin/handler/g

Reply:
I think technically the most correct term should be "package" but it's really unimportant.